packages/stable/src/cogniteClient.ts

```diff
@@ -215,7 +215,7 @@ export default class CogniteClient extends BaseCogniteClient {
       this.httpClient,
       this.metadataMap
     );
-    this.spatialApi = this.apiFactory(SpatialAPI, 'spatial');
+    this.geospatialApi = this.apiFactory(GeospatialAPI, 'geospatial');
```

Comment:
this is breaking and will require a major bump right?

Reply:
@rek 
Yes. I've added BREAKING CHANGE: description in the commit message footer.
I'm hoping just adding that to one of the commit works 🤔

Reply:
i think the important part is the message when you 'squash and merge'

Reply:
you need to rename the title to `feat` instead of `chore`.

Reply:
or perhaps just `fix`, it doesn't seem like a typical chore to do a breaking change to me

Reply:
I was wrong =D